src/test/java/com/alipay/oceanbase/rpc/ObTableClientFilterColumnTest.java

```diff
@@ -0,0 +1,335 @@
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation.ObTableAggregationResult;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.mutate.ObTableQueryAndMutateRequest;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.mutate.ObTableQueryAndMutateResult;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.syncquery.ObQueryOperationType;
+import com.alipay.oceanbase.rpc.stream.QueryResultSet;
+import com.alipay.oceanbase.rpc.stream.async.ObTableQueryAsyncStreamResult;
+import com.alipay.oceanbase.rpc.table.ObTable;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.ObObj;
+import com.alipay.oceanbase.rpc.table.ObTableClientQueryAsyncImpl;
+import com.alipay.oceanbase.rpc.table.ObTableClientQueryImpl;
+import com.alipay.oceanbase.rpc.table.ObTableParam;
+import com.alipay.oceanbase.rpc.table.api.Table;
+import com.alipay.oceanbase.rpc.table.api.TableBatchOps;
+import com.alipay.oceanbase.rpc.table.api.TableQuery;
+import com.alipay.oceanbase.rpc.threadlocal.ThreadLocalMap;
+import com.alipay.oceanbase.rpc.util.ObTableClientTestUtil;
+import com.alipay.oceanbase.rpc.util.ObTableHotkeyThrottleUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import static com.alipay.oceanbase.rpc.filter.ObTableFilterFactory.*;
+import static com.alipay.oceanbase.rpc.mutation.MutationFactory.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class ObTableClientFilterColumnTest {
+    private ObTableClient client;
+    @Before
+    public void setup() throws Exception {
+        System.setProperty("ob_table_min_rslist_refresh_interval_millis", "1");
+
+        final ObTableClient obTableClient = ObTableClientTestUtil.newTestClient();
+        obTableClient.setMetadataRefreshInterval(100);
+        obTableClient.addProperty(Property.RPC_CONNECT_TIMEOUT.getKey(), "800");
+        obTableClient.addProperty(Property.RPC_LOGIN_TIMEOUT.getKey(), "800");
+        obTableClient.addProperty(Property.SERVER_CONNECTION_POOL_SIZE.getKey(), "1");
+        obTableClient.addProperty(Property.RPC_EXECUTE_TIMEOUT.getKey(), "3000");
+        obTableClient.addProperty(Property.RUNTIME_BATCH_MAX_WAIT.getKey(), "3000");
+        obTableClient.addProperty(Property.RUNTIME_BATCH_EXECUTOR.getKey(), "32");
+        obTableClient.addProperty(Property.RPC_OPERATION_TIMEOUT.getKey(), "3000");
+        obTableClient.addProperty(Property.SERVER_ENABLE_REROUTING.getKey(), "False");
+        obTableClient.init();
+
+        this.client = obTableClient;
+    }
+
+    @Test
+    // Test aggregation with empty table
+    public void testAggregationEmptyVal() throws Exception {
```

Comment:
Move this to ObTableClientTest, or collect all aggregation test from ObTableClientTest and this file into ObTableAggregationTest

Reply:
already edit

Reply:
add ObTableAggregationTest file